src/Symfony/Component/Routing/Loader/YamlFileLoader.php

```diff
@@ -127,6 +127,24 @@ protected function parseRoute(RouteCollection $collection, $name, array $config,
         $methods = isset($config['methods']) ? $config['methods'] : array();
         $condition = isset($config['condition']) ? $config['condition'] : null;
 
+        if (isset($requirements['_method'])) {
+            if (0 === count($methods)) {
+                $methods = explode('|', $requirements['_method']);
```

Comment:
This will now priotize _method over methods option which is a BC break for people who defined both in their route definition.